libs/auth/scopes/src/lib/sjukra.scope.ts

```diff
@@ -0,0 +1,3 @@
+export enum SjukraScope {
+  europeanHealthInsuranceCard = '@sjukra.is/sjukratryggingakort',
```

Comment:
Does this scope exist in all environments? Should be assigned to the application-system machine client in all envs and to the local dev client in the dev environment?

Reply:
Yes, as I understand it, this is the scope that will be used in all environments as this is the scope the access token is assigned and is verified by sjúkrastofnun API

Reply:
But can you confirm that this scope exists and is assigned to the right clients in all environments? If not it might break the release.